sources/editor/Stride.Assets.Presentation/AssetEditors/ProjectWatcher.cs

```diff
@@ -350,25 +375,24 @@ private async Task<bool> UpdateProject(TrackedAssembly trackedAssembly)
 
             return true;
         }
-
+        private Solution solution;
```

Comment:
Unused

Reply:
https://github.com/stride3d/stride/pull/2106/commits/0f4d84f41cb851c448879694468ce8c04d567f09
removed it